python/doc/theory/reliability_sensitivity/ranking_pcc.rst

```diff
@@ -13,15 +13,17 @@ and the different components :math:`X_i`.
 The basic method of hierarchical ordering using Pearson’s coefficients
 deals with the case where the variable :math:`Y` linearly
 depends on :math:`\inputDim` variables
-:math:`\left\{ X_1,\ldots,X_{\inputDim} \right\}` but this can be misleading
-when statistical dependencies or interactions between the variables
-:math:`X_i` (e.g. a crossed term :math:`X_i \times X_j`) exist. In such
-a situation, the partial correlation coefficients can be more useful in
-ordering the uncertainty hierarchically: the partial correlation
-coefficients :math:`\textrm{PCC}_{X_i,Y}` between the variables
-:math:`Y` and :math:`X_i` attempts to measure the residual influence
+:math:`\left\{ X_1,\ldots,X_{\inputDim} \right\}`.
```

Comment:
I would also fix the first sentence from lines 6 to 9:
```
Partial Correlation Coefficients deal with analyzing the influence of the random vector :math:`\inputRV = \left( X_1,\ldots,X_{\inputDim} \right)` on a random variable :math:`Y`, which is being studied.
```
The second sentence from lines 9 to 11 can also be improved:
```
Here we attempt to measure the linear relationships that exist between :math:`Y` and the different components :math:`X_i`.
```
The third sentence can also be improved:
```
The basic method of hierarchical ordering using Pearson’s coefficients deals with the case where the variable :math:`Y` depends linearly on the :math:`\inputDim` variables :math:`\left{ X_1,\ldots,X_{\inputDim} \right}`.
```